README.rst

```diff
@@ -4,10 +4,12 @@ MeshPy: Simplicial Mesh Generation from Python
 .. image:: https://gitlab.tiker.net/inducer/meshpy/badges/main/pipeline.svg
     :alt: Gitlab Build Status
     :target: https://gitlab.tiker.net/inducer/meshpy/commits/main
+
```

Comment:
Why that? It seems that would put them on different lines, which is not what I'd like.

Reply:
Oops! Reverted in latest version.